From 162cee83ba98abb6ee21bb270ff0c6a09e97cd1f Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Tue, 10 Feb 2015 20:32:02 -0800
Subject: [PATCH 1/2] darwin: build fix

../../../src/mesa/main/compiler.h:47:10: fatal error: 'util/macros.h' file not found

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 1c67a5687a35e984323b6034adb914a66d64bb2f)
---
 src/glx/apple/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/glx/apple/Makefile.am b/src/glx/apple/Makefile.am
index 894ab5b..8d03225 100644
--- a/src/glx/apple/Makefile.am
+++ b/src/glx/apple/Makefile.am
@@ -1,6 +1,7 @@
 noinst_LTLIBRARIES = libappleglx.la
 
 AM_CFLAGS = \
+	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/include \
 	-I$(top_srcdir)/src/glx \
 	-I$(top_srcdir)/src/mesa \
-- 
2.3.0

